### PR TITLE
Fix accidental email exposure

### PR DESCRIPTION
**Pull request by hexylena**
cc @jmchilton

### CODE REVIEW

File: templates/webapps/galaxy/history/share.mako [hunk 1]
```diff
@@ -38,8 +38,14 @@
...
                     <% existing_emails = ','.join([ d.user.email for d in history.users_shared_with ]) %>
                     <label>Galaxy user emails with which to share histories</label>
+                    %if trans.app.config.expose_user_email or trans.app.config.expose_user_name or trans.user_is_admin():
                     <input type="hidden" id="email_select" name="email" value="${ existing_emails }" style="float: left; width: 250px; margin-right: 10px;">
                     </input>
+                    %else:
+                    <input type="text" name="email" value="${ existing_emails }" size="40">
+                    </input>
+                    %endif
+                    </input>
```
Comment (martenson) [inline templates/webapps/galaxy/history/share.mako]: extra `</input>` ?

Comment (hexylena) [reply templates/webapps/galaxy/history/share.mako]: Do they not need trailing slashes

Comment (hexylena) [reply templates/webapps/galaxy/history/share.mako]: Oh, wow, you.re right. Loads/functions, just invalid.

**Comment by martenson** [comment]
xref https://github.com/galaxyproject/galaxy/issues/3372

**Review by jmchilton — APPROVED** [review]
Looks good to me and works in my testing.

**Pull request merged**